### DISCUSSION TITLE
Question about biome size

### DISCUSSION THREAD

**Discussion by AmibeSkyfy16** [misode/misode.github.io#183] [Q&A] [answered]
Is it possible to create a Minecraft map with a larger biome size. Because i find that normal generation create biome size too small and maps generate with "Large Biome" parameter are too big. I would like to find a compromise between normal and large biome.

**Comment by misode** [comment, marked as answer]
In the latest snapshot those settings have been moved to the `data/minecraft/worldgen/noise` folder. If you go to that folder and find the `temperature`, `humidity`, etc files, you can change the firstOctave by one (for example `-9` -> `-10`), this should double the scale of all biomes. Note that these settings can currently not be changed per dimension.

**Reply by AmibeSkyfy16** [reply]
Thanks, i will try this, i think its the solution for my problem
